src/components/AddBook.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default function AddBook() {
+  return (
+    <>
+      <h3>ADD NEW BOOK</h3>
+      <form>
+        <input type="text" placeholder="Book Title" />
+        <select key="author">
+          <option value="Category">all authors</option>
+        </select>
+        <button type="submit">Add Book</button>
```

Comment:
- [ ] Please, consider that you have to implement two inputs: title and author. In your version, I can see only one input for the Title and the dropdown list for authors. Here should be the input for Author, instead of the dropdown list. 👍